src/DynamoCoreWpf/Views/GuidedTour/PopupWindow.xaml.cs

```diff
@@ -77,9 +77,7 @@ private void PopupWindow_Opened(object sender, EventArgs e)
             {
                 ContentRichTextBox.Visibility = Visibility.Hidden;
 
-                if(webBrowserWindow == null)
-                    webBrowserWindow = new WebBrowserWindow(popupViewModel, hostControlInfo);
-
+                webBrowserWindow = new WebBrowserWindow(popupViewModel, hostControlInfo);
```

Comment:
Dynamo was crashing when a step related to the webbrowser was opened for the second time. 

![packagesTourCrash](https://user-images.githubusercontent.com/89042471/141831864-931b4081-2e94-432e-b531-16f132f0162a.gif)

It was happening because the object was losing its properties after it was closing, and it was not finding `UpdatePosition`  method in this line.

![image](https://user-images.githubusercontent.com/89042471/141832784-dafc1dcd-677b-459b-98fa-a39dfeb49abe.png)

To update its properties I removed the verification to instantiate the webbrowser window when necessary.